### PR TITLE
Document where to find container images for Kubernetes components

### PR DESCRIPTION
**Pull request by afro-coder**
Hi,

Description: Guide users where to find/download Kubernetes component container images. This will help users locate/download Kubernetes container images i.e Api Server, Kube-proxy. Also useful for users trying to deploy to air-gapped systems.

Closes: #31293 

Reference Page: https://kubernetes.io/docs/setup/

Question: Currently we are moving to [registry.k8s.io](https://registry.k8s.io/) however this returns a 404 is it okay to keep it for now or should I change it to [k8s.gcr.io](https://console.cloud.google.com/gcr/images/k8s-artifacts-prod/GLOBAL)

image

Signed-off-by: afro-coder <leon9923@gmail.com>

Regards,
Leon.

### CODE REVIEW

File: content/en/docs/setup/_index.md [hunk 1]
```diff
@@ -25,7 +25,16 @@ When you install Kubernetes, choose an installation type based on: ease of maint
...
+on a local machine, into the cloud, or for your own datacenter. 
+
+[Kubernetes components](/docs/concepts/overview/components/) i.e Api Server, kube-proxy can also be 
+deployed as [container images](/releases/download/#container-images) 
+within the cluster, these come handy if you're installing Kubernetes in an air-gapped system, storing them
+in a local repository, or downloading them on a usb drive. It is recommended to run the components
+as container images.
+
+These images are stored on https://registry.k8s.io and can be pulled/downloaded based 
+on the architecture and version.
```
[the remark this answers is not in the document]
Comment (afro-coder) [reply content/en/docs/setup/_index.md:37]: Sure I'll rewrite it with the style guide and remove those details, I was just trying to give a hint to the reader as to why these may be required, etc like an example for why container images could be used.

File: content/en/docs/setup/_index.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ control, available resources, and expertise required to operate and manage a clu
 You can [download Kubernetes](/releases/download/) to deploy a Kubernetes cluster
 on a local machine, into the cloud, or for your own datacenter.
 
+[Kubernetes components](/docs/concepts/overview/components/) such as `API Server` or `Kube-proxy` can also be
```
Comment (sftim) [inline content/en/docs/setup/_index.md:30]:
```suggestion
Several [Kubernetes components](/docs/concepts/overview/components/) such as `kube-apiserver` or `kube-proxy` can also be
```

**Review by sftim — CHANGES_REQUESTED** [review]
It's close but, for me, another tweak's needed. Sorry about that.

File: content/en/docs/setup/_index.md (hunk 2 above)
Comment (afro-coder) [reply content/en/docs/setup/_index.md:30]: No worries this seems better I'll update these!

File: content/en/docs/setup/_index.md [hunk 3]
```diff
@@ -27,6 +27,11 @@ control, available resources, and expertise required to operate and manage a clu
...
+deployed as [container images](/releases/download/#container-images) within the cluster.
+
+It is **recommended** to run Kubernetes Components as container images.
```
[the remark this answers is not in the document]
Comment (afro-coder) [reply content/en/docs/setup/_index.md:33]: Makes sense, I'll update these!

**Pull request merged**